test/sql/hnsw_cost_estimate.sql

```diff
@@ -1,75 +1,61 @@
-DROP INDEX empty_idx;
+CREATE TABLE sift_base10k_0 (
+     id SERIAL PRIMARY KEY,
+     v real[128]
+);
+CREATE INDEX hnsw_idx ON sift_base10k_0 USING hnsw (v dist_l2sq_ops) WITH (M=2, ef_construction=10, ef=2, dim=2);
+SET _lantern_internal.is_test = true;
+EXPLAIN (ANALYZE,TIMING FALSE) SELECT * FROM sift_base10k_0 order by v <-> '{1, 2}'
+LIMIT 10;
+SET _lantern_internal.is_test = false;
+DROP INDEX hnsw_idx;
 
 -- Case 1, more data in index.
 -- Should see higher cost than Case 0.
-CREATE INDEX hnsw_idx ON sift_base10k USING hnsw (v dist_l2sq_ops) WITH (M=2, ef_construction=10, ef=4, dim=128);
-SELECT is_cost_estimate_within_error(format(:'explain_query_template', :'v4444'), 3.00);
+CREATE TABLE sift_base10k_1 (
+     id SERIAL PRIMARY KEY,
+     v real[128]
+);
+\copy sift_base10k_1 (v) FROM '/tmp/lantern/vector_datasets/siftsmall_base_arrays.csv' with csv;
+CREATE INDEX hnsw_idx ON sift_base10k_1 USING hnsw (v dist_l2sq_ops) WITH (M=2, ef_construction=10, ef=4, dim=128);
+SELECT V AS v4444  FROM sift_base10k_1 WHERE id = 4444 \gset
+SET _lantern_internal.is_test = true;
+EXPLAIN (ANALYZE,TIMING FALSE) SELECT * FROM sift_base10k_1 order by v <-> :'v4444'
+LIMIT 10;
+SET _lantern_internal.is_test = false;
 DROP INDEX hnsw_idx;
 
 -- Case 2, higher M.
 -- Should see higher cost than Case 1.
-CREATE INDEX hnsw_idx ON sift_base10k USING hnsw (v dist_l2sq_ops) WITH (M=20, ef_construction=10, ef=4, dim=128);
-SELECT is_cost_estimate_within_error(format(:'explain_query_template', :'v4444'), 3.27);
+CREATE TABLE sift_base10k_2 (
+     id SERIAL PRIMARY KEY,
+     v real[128]
+);
+\copy sift_base10k_2 (v) FROM '/tmp/lantern/vector_datasets/siftsmall_base_arrays.csv' with csv;
+CREATE INDEX hnsw_idx ON sift_base10k_2 USING hnsw (v dist_l2sq_ops) WITH (M=20, ef_construction=10, ef=4, dim=128);
+SELECT V AS v4444  FROM sift_base10k_2 WHERE id = 4444 \gset
+SET _lantern_internal.is_test = true;
+EXPLAIN (ANALYZE,TIMING FALSE) SELECT * FROM sift_base10k_2 order by v <-> :'v4444'
+LIMIT 10;
+SET _lantern_internal.is_test = false;
 DROP INDEX hnsw_idx;
 
 -- Case 3, higher ef.
 -- Should see higher cost than Case 2.
-CREATE INDEX hnsw_idx ON sift_base10k USING hnsw (v dist_l2sq_ops) WITH (M=20, ef_construction=10, ef=16, dim=128);
-SELECT is_cost_estimate_within_error(format(:'explain_query_template', :'v4444'), 3.91);
+CREATE TABLE sift_base10k_3 (
+     id SERIAL PRIMARY KEY,
+     v real[128]
+);
+\copy sift_base10k_3 (v) FROM '/tmp/lantern/vector_datasets/siftsmall_base_arrays.csv' with csv;
+CREATE INDEX hnsw_idx ON sift_base10k_3 USING hnsw (v dist_l2sq_ops) WITH (M=20, ef_construction=10, ef=16, dim=128);
+SELECT V AS v4444  FROM sift_base10k_3 WHERE id = 4444 \gset
+SET _lantern_internal.is_test = true;
+EXPLAIN (ANALYZE,TIMING FALSE) SELECT * FROM sift_base10k_3 order by v <-> :'v4444'
+LIMIT 10;
+SET _lantern_internal.is_test = false;
```

Comment:
I think this file was also changed by accident when rebasing. There should be no changes other than `SET _lantern_internal.is_test = false;`

Reply:
My bad I must have made a mistake while rebasing. fixing it now